services/QuillLMS/config/routes.rb

```diff
@@ -738,7 +738,8 @@
     collection do
       post :count
       get :options
-      post :top_x
```

Comment:
I would double check with @anathomical about this, but I think we do need this to be a `post` (and that the `data_export` call may need to be one too due to the length of the params).

Reply:
I checked with him and you're right, I'll change this to a post request